jenkins-remoting.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: jenkins-remoting
+  version: 3206.vb_15dcf73f6a_9
+  epoch: 0
+  description: Jenkins Remoting module
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - maven
+      - openjdk-21
+      - openjdk-21-default-jvm
+      - wolfi-base
+      - wolfi-baselayout
+  environment:
+    LANG: en_US.UTF-8
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jenkinsci/remoting.git
+      tag: ${{package.version}}
+      expected-commit: b15dcf73f6a9b71da045bc471697702d99e871af
+
+  - name: build remoting libary jar
+    runs: |
+      export MAVEN_OPTS="-DskipTests=true -Dspotbugs.skip=true"
+      mvn clean package
+      mkdir -p ${{targets.destdir}}/usr/share/jenkins/
+      cp target/remoting-999999-SNAPSHOT.jar ${{targets.destdir}}/usr/share/jenkins/agent.jar
+subpackages:
+  - name: jenkins-inbound-agent
+    description: Jenkins agent which can connect to Jenkins using TCP or Websocket protocols
+    pipeline:
+      - uses: git-checkout
+        with:
+          repository: https://github.com/jenkinsci/docker-agent.git
```

Comment:
this is a different repo. though 2 packages are related, it's best to split it?

jenkins-inbound-agent will have remoting + busybox + java as runtime deps. wdyt

Reply:
Thanks for spotting this @tuananh

Reply:
that's good, let me break them